indexer/aggregates_indexer.go

```diff
@@ -0,0 +1,62 @@
+package indexer
+
+import (
+	"context"
+
+	dbv1 "api.audius.co/api/dbv1"
+	"api.audius.co/config"
+	"api.audius.co/database"
+	"api.audius.co/jobs"
+	"api.audius.co/logging"
+	"github.com/jackc/pgx/v5/pgxpool"
+	"go.uber.org/zap"
+)
+
+type AggregatesIndexer struct {
```

Comment:
Extremely nitpicky, but i wonder if we should keep the word "indexer" for indexing external data vs aggregating or reprocessing data in the db?

Reply:
Yeah you're probably right. I will update that before I merge.